src/shared/compatibility.js

```diff
@@ -16,6 +16,9 @@
 
 import { isNodeJS } from "./is_node.js";
 
+// GlobalThis object is available since chrome 71.
+require("core-js/stable/global-this.js");
+
```

Comment:
Please note that we do *not* support any browser/environment that lack native `globalThis` support; please refer to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis#browser_compatibility and https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support

Hence this is unfortunately not an acceptable change here, and note that in general we also don't allow bundling of compatibility code *unconditionally* like that either.

Reply:
OK, thanks for the information. BrightSign uses a Chrome 69 so indeed there is no globalThis. I ask them if they can upgrade their Chrome.

Reply:
> BrightSign uses a Chrome 69 so indeed there is no globalThis.

In that case, as mentioned above, we will *not* support that environment.

Also, given that Chrome 69 was release on 2018-09-04 (according to https://en.wikipedia.org/wiki/Google_Chrome_version_history) it may contain *known* security bugs and no one ought to be using such an old browser any more. (I'm really not convinced that we want to support an environment that's so bad about updating their software.)